doc/api/path.md

```diff
@@ -416,6 +416,16 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+### path.posix.normalize(path)
```

Comment:
I think this note belongs in the `normalize()` documentation, not as a different subsection in `path.posix`.